SU2_CFD/src/solver_direct_turbulent.cpp

```diff
@@ -3608,8 +3611,8 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
     
     kine  = nodes->GetSolution(iPoint,0);
     omega = nodes->GetSolution(iPoint,1);
-    zeta = min(1.0/omega, a1/(strMag*F2));
-    muT = min(max(rho*kine*zeta,0.0),1.0);
+    zeta  = min(1.0/omega, a1/(Omega*F2));
```

Comment:
This is technically a different version of the SST model.  For reference, the old version was SST-2003 and the new version is SST on [NASA LaRC's TMR model page](https://turbmodels.larc.nasa.gov/sst.html).  The paper cited on SST-2003 doesn't provide justification for the differences between versions.

There are actually a number of changes between SST and SST-2003.  I reviewed the changes I could find (model constants, production limiter, and CD_kw), and everything in the code now seem to match SST (and not SST-2003).  So I think you're consistent.

Is there a particular reason you chose the SST version over the SST-2003 version?

Reply:
This one is more straightforward.. I think consistency is the most important item here. We advertise the model as the "standard" SST model, and so it should be. As you know, folks take the differences in the models seriously, at least in the aeronautics world, which is important for expectations and reproduction of results. Seems to me that we only had a mismatch in the one term.

I think it would be great to also have more variants of the SST model, like we do for SA, since I think the later variants have become more commonly used, at least in some commercial codes. As long as we implement and catalog them as they appear on the TMR, they would be nice additions.